src/screens/Home/components/ChooseLocation/components/MapLabel.js

```diff
@@ -1,38 +1,72 @@
 import React from 'react'
-import { Anchor, Box } from 'grommet'
+import { Anchor, Box, Image } from 'grommet'
 import styled from 'styled-components'
 import { func, shape, string } from 'prop-types'
+import { withResponsiveContext } from '@zooniverse/react-components'
 
 export const StyledAnchor = styled(Anchor)`
-  padding: 0.5em;
   white-space: nowrap;
+  position: relative;
+  text-decoration: none;
 
-  :focus, :hover {
-    background: white;
-    text-decoration: underline;
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 100%;
+    height: 1px;
+    background: #113e3b;
+  }
+
+  &:hover {
+    text-decoration: none;
+  }
+`
+
+const Relative = styled(Box)`
+  position: relative;
+  
+  &:hover {
```

Comment:
this adds the line from anchor text to thumbnail image? 

I think can send `mobile` prop and conditionally render, omitting for small mobile view. maybe similar to https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/plugins/tasks/SurveyTask/components/components/Chooser/components/Choices/components/ChoiceButton.js#L12

Reply:
Yes, it's styled so the container's bottom border can extend seamlessly to the preview thumbnail image on hover, but i agree it's not needed on mobile, so i'll change that!